### ISSUE TITLE
Allow parallel generation of answers from multiple models

### ISSUE THREAD

**Issue by dit7ya** [n3d1117/chatgpt-telegram-bot#315]
### Use case/ Motivation

I have the bot set-up with the GPT-4 model which is qualitatively better than GPT-3. However, GPT-4 is much slower than GPT-3. 

For some simple queries, I just want a fast answer from GPT-3 because both the answers are usually same.
For some queries, I am not sure, so I just want to send the message to both the models and start receiving parallel answers. If I am satisfied with GPT-3's answer I will just move on with my work - if not I want to wait for GPT-4 to finish.

**Comment by n3d1117** [comment]
Hi @dit7ya, that would be cool but I feel like it's out of scope for this project. How would you decide which model to use based on the query anyway?

**Comment by dit7ya** [comment]
By some sort of commands I suppose?

no command -> the default model answers
`/3` or `/4` -> the other model answers
`/both` -> both models answer in parallel
